From f75689f99e6bbe7336493f86c470f85879f2663d Mon Sep 17 00:00:00 2001
From: Tobias Jungel <tobias.jungel@bisdn.de>
Date: Thu, 11 Apr 2019 21:16:23 +0200
Subject: [PATCH 08/12] ag8032: fix -Werror=unused-result

errors like:
platform_lib.c:302:6: error: ignoring return value of system, declared
with attribute warn_unused_result [-Werror=unused-result]
|       system (psu->pmbus_insert_cmd);
|       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
---
 .../onlp/builds/src/module/src/platform_lib.c               | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/packages/platforms/delta/x86-64/x86-64-delta-ag8032/onlp/builds/src/module/src/platform_lib.c b/packages/platforms/delta/x86-64/x86-64-delta-ag8032/onlp/builds/src/module/src/platform_lib.c
index e5755423..8495356c 100644
--- a/packages/platforms/delta/x86-64/x86-64-delta-ag8032/onlp/builds/src/module/src/platform_lib.c
+++ b/packages/platforms/delta/x86-64/x86-64-delta-ag8032/onlp/builds/src/module/src/platform_lib.c
@@ -299,7 +299,7 @@ int plat_psu_state_update (plat_psu_t *psu)
 
 				psu->state = PLAT_PSU_STATE_PMBUS_READY;
 				if (!plat_os_file_is_existed(psu->pmbus_ready_path) && psu->pmbus_insert_cmd) {
-					system (psu->pmbus_insert_cmd);
+					(void)!system (psu->pmbus_insert_cmd);
 				}
 				if (psu->event_callback)
 					psu->event_callback(psu, PLAT_PSU_PMBUS_CONNECT);
@@ -311,7 +311,7 @@ int plat_psu_state_update (plat_psu_t *psu)
 			if (!present) {
 				psu->state = PLAT_PSU_STATE_UNPRESENT;
 				if (psu->pmbus_remove_cmd) {
-					system (psu->pmbus_remove_cmd);
+					(void)!system (psu->pmbus_remove_cmd);
 				}
 				if (psu->event_callback)
 					psu->event_callback(psu, PLAT_PSU_EVENT_UNPLUG);
@@ -323,7 +323,7 @@ int plat_psu_state_update (plat_psu_t *psu)
 
 				psu->state = PLAT_PSU_STATE_PRESENT;
 				if (psu->pmbus_remove_cmd) {
-					system (psu->pmbus_remove_cmd);
+					(void)!system (psu->pmbus_remove_cmd);
 				}
 				if (psu->event_callback)
 					psu->event_callback(psu, PLAT_PSU_PMBUS_DISCONNECT);
-- 
2.20.1

